CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,9 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2022-03-22 New Datetime formatter in the File Importer
+<p>New Datetime formatter 'EXCEL' is added as an option for parsing native excel datetime into Constellation which is available as an 'Datetime Format' in the 'Datetime Parameters' dialog when the DateTime attribute is right clicked and 'Formatter' -> 'Datetime' is choosen.</p>
+
```

Comment:
I think I would split up this sentence a little. Maybe something like "A new Datetime formatter has been added to the File Importer for parsing native Excel datetimes into Constellation. You can use it by right-clicking an attribute, navigating to Formatter -> Datetime and choosing 'EXCEL' from the format dropdown list"

Reply:
Yes it' much better. Changed now.